<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QComboBox" name="filenameComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="maxVisibleItems">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="removeCurrent">
     <property name="toolTip">
      <string>remove Current entry from Combo Box</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="Experiment.qrc">
       <normaloff>:/openicon/icons/edit-clear-locationbar-rtl.png</normaloff>:/openicon/icons/edit-clear-locationbar-rtl.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="newButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="Experiment.qrc">
       <normaloff>:/other/icons/document-new-icon.png</normaloff>:/other/icons/document-new-icon.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="openButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="Experiment.qrc">
       <normaloff>:/openicon/icons/document-open-5.png</normaloff>:/openicon/icons/document-open-5.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="saveButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="Experiment.qrc">
       <normaloff>:/kde/icons/filesave.png</normaloff>:/kde/icons/filesave.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="resetButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="Experiment.qrc">
       <normaloff>:/openicon/icons/document-revert-5.png</normaloff>:/openicon/icons/document-revert-5.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/kde/icons/document-open-5.png</normaloff>:/kde/icons/document-open-5.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/kde/icons/filesave.png</normaloff>:/kde/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/document-revert-5.png</normaloff>:/openicon/icons/document-revert-5.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset to file on disk</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/other/icons/document-new-icon.png</normaloff>:/other/icons/document-new-icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Experiment.qrc"/>
 </resources>
 <connections/>
</ui>
